stdlib/public/Concurrency/CheckedContinuation.swift

```diff
@@ -251,6 +271,18 @@ public func withCheckedContinuation<T>(
   }
 }
 
+/// Suspends the current task,
+/// then calls the given closure with a checked throwing continuation for the current task.
```

Comment:
oh wait, that's not quite right.

We don't unconditionally suspend here do we?

This is just a potential suspension point and if it is immediately resumed we never suspended, or did I get that wrong @jckarter ?

Reply:
From our discussion, we'll dig into this detail and fix separately if this wording isn't correct.

Reply:
Okey it seems the wording is correct :)